app/routes/events/components/EventEditor/EditorSection/Header.tsx

```diff
@@ -48,7 +48,7 @@ const Header = ({
       <Field
         label="Tittel"
         name="title"
-        placeholder="Tittel"
+        placeholder="Sjakkrave"
```

Comment:
Sneaky marketing

Reply:
oh no you found it! 😆

Reply:
I really dislike when the label is the same as the placeholder

Reply:
Absolutely agree(: